crates/store/src/db/models/queries/accounts.rs

```diff
@@ -929,11 +867,11 @@ pub(crate) fn upsert_accounts(
             account_commitment: update.final_state_commitment().to_bytes(),
             block_num: block_num.to_raw_sql(),
             nonce: full_account.as_ref().map(|account| nonce_to_raw_sql(account.nonce())),
-            storage: full_account.as_ref().map(|account| account.storage().to_bytes()),
-            vault: full_account.as_ref().map(|account| account.vault().to_bytes()),
             code_commitment: full_account
                 .as_ref()
                 .map(|account| account.code().commitment().to_bytes()),
+            storage_header: full_account.as_ref().map(|account| account.storage().to_bytes()),
```

Comment:
I think this answers a few of my comments (including the previous one): even though we've renamed the field from `storage` to `storage_header`, we are still saving the full account storage into it.